demo/midgard-node/src/database/address.ts

```diff
@@ -0,0 +1,124 @@
+import { Database } from "@/services/database.js";
+import { SqlClient, SqlError } from "@effect/sql";
+import { Effect } from "effect";
+import { mapSqlError } from "@/database/utils/common.js"
+import { Address, CML } from "@lucid-evolution/lucid";
+import * as MempoolDB from "@/database/mempool.js"
+import * as ImmutableDB from "@/database/immutable.js"
+import * as Tx from "@/database/utils/tx.js"
+import * as Ledger from "@/database/utils/ledger.js"
+import { MempoolLedgerDB } from "./index.js";
+
+const tableName = "address"
```

Comment:
If you decide to go with `addressHistory.ts` for the filename, it'll probably better to rename the table as well:
```suggestion
const tableName = "address_history";
```